src/opcuabridge/opcuabridgeclient.cc

```diff
@@ -163,14 +165,25 @@ VersionReport Client::recvVersionReport() const {
   return version_report;
 }
 
+OriginalManifest Client::recvOriginalManifest() const {
+  OriginalManifest original_manifest;
+  if (UA_Client_getState(client_) != UA_CLIENTSTATE_DISCONNECTED) {
+    original_manifest.ClientRead(client_);
+  }
+  return original_manifest;
+}
+
 bool Client::sendMetadataFiles(std::vector<MetadataFile>& files) const {
   MetadataFiles metadatafiles;
   bool retval = true;
   if (UA_Client_getState(client_) != UA_CLIENTSTATE_DISCONNECTED) {
+    std::srand(std::time(nullptr));
```

Comment:
it is very slow and should be run only once, but You run it every time you send metadata files

Reply:
if You run this method on primary You should not run srand because  because we already call it at aktualizr startup.